### PR TITLE
Ads clean code

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: pilot/pkg/proxy/envoy/v2/ads.go [hunk 1]
```diff
@@ -266,7 +209,6 @@ func (s *DiscoveryServer) StreamAggregatedResources(stream ads.AggregatedDiscove
 	reqChannel := make(chan *xdsapi.DiscoveryRequest, 1)
 	go receiveThread(con, reqChannel, &receiveError)
 
-	node := &core.Node{}
```
Comment (howardjohn) [inline pilot/pkg/proxy/envoy/v2/ads.go]: @kyessenov please take a look I'm not sure you if specifically used this rather than con.node for a reason?

Comment (hzxuzhonghu) [reply pilot/pkg/proxy/envoy/v2/ads.go]: This is a legacy var, @kyessenov just modify based on it.

**Pull request merged**